app/assets/javascripts/reviews.js

```diff
@@ -1,6 +1,14 @@
 var ready;
 
 ready = function() {
+	$('.myreview-container').parent().css({
+		'box-shadow': 'none',
+		'background-color': '#eee',
+  		'padding-top': '0px',
+  		'padding-left': '0px',
+  		'padding-right': '0px',
+	});
```

Comment:
why are you trying to change `#content-container` styling then only on this one page?